pkgs/applications/networking/instant-messengers/signald/deps.json

```diff
@@ -0,0 +1,689 @@
+{
+ "https://gitlab.com/api/v4/groups/6853927/-/packages/maven/org": {
+  "signald#libsignal-client-aarch64-apple-darwin/0.21.1": {
+   "jar": "GZyZBUokXraIAXXRoGMFoU48FzUym80J0XU2o/fbEgE=",
+   "module": "Z6k9Pt5ac5il2uK2ngQn0Mc8utSRChIl+pO7UeOmbJs=",
+   "pom": "b2SOa4rB+2SGAohZpsv7ZNUKPrcCBIONBiflsaLJA7A="
```

Comment:
If those are supposed to be nix sri hashes, then they are missing the prefix.

Reply:
I'm using sha256 rather than sri to keep the lockfiles slightly smaller, and because the URLs are all generated in the same way anyway.

Reply:
Nix SRI hashes have the exact same length but with a small prefix denoting which hash was used:

`sha256-jZkUKv2SV28wsM18tCqNxoCZmLxdYH2Idh9RLibH2yA=`

vs.

`GZyZBUokXraIAXXRoGMFoU48FzUym80J0XU2o/fbEgE=`

Each of these prefixes should be about a single bit large with Huffman coding, so that's not really significant. Anyone who cares about file size to this degree will be using compression.

Reply:
I have nothing against sri hashes in principle, I just don't see a reason to use them either

Reply:
The primary upsides are that it's a standard almost everyone in the Nix community knows, that tools can work with and that have certain nice properties such as very directly declaring which hash function is used.